### PR TITLE
Setting up redux with redux toolkits

### PR DESCRIPTION
**Pull request by jamespure**

### CODE REVIEW

**Review by rufuller — COMMENTED** [review]
🙂

File: src/redux/store.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { configureStore } from "@reduxjs/toolkit";
+
+export const store = configureStore({
+  reducer: {},
```
Comment (lindentree) [inline src/redux/store.ts:4]: I would strongly recommend separating out any initial reducer into its own file, and coming up with some placeholder value for the initial state, following the examples here: https://redux.js.org/tutorials/fundamentals/part-4-store

Ideally the initial state is latitude/longitude, so you can hardcode some random place in the US. Then another good step would be to verify that the screen components can read that value from the store, maybe by using alert/console.log on render.

**Review by lindentree — CHANGES_REQUESTED** [review]
Need verification that redux is set up properly and the store values can be read by a component.

File: src/redux/store.ts (hunk 1 above)
Comment (rweitzman) [reply src/redux/store.ts:4]: I agree! @jamespure I have some files I can share for suggestions if you would like!

Comment (jamespure) [reply src/redux/store.ts:4]: @lindentree @rweitzman This is redux toolkit. Here a ref to the docs https://redux-toolkit.js.org/usage/usage-guide

You'll add create slices in the features folder and add the slice to the store reducer.

Comment (lindentree) [reply src/redux/store.ts:4]: Initial state is still a core part of this feature/ticket and should not be left for later. The documentation for store setup isn’t significantly different: https://redux-toolkit.js.org/tutorials/typescript

**Comment by HotSoupRepos** [comment]
@jamespure we are waiting on you to resolve this PR!

**Pull request closed without merging**